site/content/blog/2021/2021-11-02-ADRs-Das-Format.adoc

```diff
@@ -0,0 +1,88 @@
+= Architecture Decision Records Teil 1 - Warum Entscheidungen dokumentieren?
+:jbake-date: 2021-04-11
+:jbake-author: jdienst
+:jbake-type: post
+:jbake-toc: true
+:jbake-status: published
+:jbake-tags: asciidoc, softwarearchitecture, architecture decision records
+:doctype: article
+:toc: macro
+
+
+Entscheidungen zu dokumentieren stellt meiner Erfahrung nach Verbindlichkeit und Transparenz her.
+Um Architekturentscheidungen zu dokumentieren hat sich für mich ein Format namens _Architecture Decision Record_ (kurz ADR) bewährt.
+
+Im Rahmen dieser Artikelserie betten wir die Architecture Decision Records in den größeren Rahmen der Architekturarbeit ein.
+Dabei gehe ich insbesondere auf den Kontext des Dokumentationstemplates arc42 und dessen Einsatz im DevOps-Umfeld ein.
+
+Wir sehen uns dazu auch intensiver die Technik an, mit der dokumentiert wird. Hier kommen AsciiDoc, Microsites und Git zum Einsatz.
+
+Aber starten wir zuerst mit dem _Warum_. Warum ist es eine gute Idee, Entscheidungen zu dokumentieren?
+
+== Undokumentierte Entscheidungen - Ein Problem auf unterschiedlichen Ebenen
+
+Undokumentierte Entscheidungen sind oft die Regel. Meistens bleiben sie eine lange Zeit unbemerkt oder kommen nie wieder zur Sprache.
+Interessant wird es, wenn an einer Stelle die Frage gestellt wird: *Warum* ist das *so*?
```

Comment:
Vorschlag:
Dies passiert meistens, wenn nach langer Zeit neue Ideen in das Projekt kommen 
und man sich fragt "warum haben wir das damals so entschieden? Können wir es jetzt nicht mit Ansatz X oder Technologie Y besser machen?".
Aber dann kann sich meist niemand mehr an die genauen Gründe erinnern.